src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/controllers/CreateReferralController.java

```diff
@@ -179,6 +180,11 @@ public ResponseEntity<CCDCallbackResponse> aboutToSubmitReferralDetails(
             )
         );
 
+        log.info("Event: Referral Email sent. "
+            + "EventId: " + ccdRequest.getEventId() + "."
+            + "Emailed to: " + caseData.getReplyToEmailAddress() + " at : " + DateTime.now()
+            + " and sent by " + userDetails.getName() + ".");
```

Comment:
we shouldn't mention the email address and user name as it's personal information. We shouldn't be have names in our logs

Reply:
Fair point.